jep/0000/README.adoc

```diff
@@ -0,0 +1,329 @@
+= JEP-0000: Sensible Permissions
+:toc: preamble
+:toclevels: 3
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+.**JEP Template**
+[TIP]
+====
+In this document, all text in a "Tip" block (or inline text with with a ":bulb:" on either side)
+MUST be removed and/or replaced with text appropriate to this JEP before submission.
```

Comment:
@mikecirioli  
I assume you will do this before submitting the PR for review?

Reply:
yes, there is still a bit of work to be done before i take this PR out of draft state.